packages/astro/src/@types/astro.ts

```diff
@@ -2336,3 +2371,25 @@ declare global {
 		'astro-dev-overlay-highlight': DevOverlayHighlight;
 	}
 }
+
+export interface PrefetchOptions {
```

Comment:
I honestly have no idea about this, because when I edit docs in this file, I don't remember ever having to edit them twice before? I am very bad at JSDoc. Someone help.

If these should show up twice, then I guess they should say the same thing? But this seems suspect to me.

Reply:
Some appeared twice like `server.host` too, and presumably the docs scanner is only able to scan `AstroUserConfig` and not the other types it depends on.

But I went ahead to fix this since it doesn't look like we need a special type for `prefetch`. I inlined the type into `AstroUserConfig` so we only have to document it once.

Reply:
I trust you!  💜